pkg/standalone/bundle.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+const bundleDetailsFileName = "details.json"
```

Comment:
Is this file as part of spec/understanding between cli and bundle installer? Or there is a probability of the name change?

Reply:
part of the spec/understanding ... see this change ... https://github.com/dapr/installer-bundle/pull/3

Reply:
Once that PR is merged, do you want to add link to that as comment? Will be helpful for code-viewers to get the context.